lib/async_storage.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
```

Comment:
I think this file is obsolete as content was transferred to async_hooks.js

Reply:
Indeed!